internal/start/start.go

```diff
@@ -555,7 +555,9 @@ EOF
 
 			"GOTRUE_SMTP_HOST=" + utils.InbucketId,
 			"GOTRUE_SMTP_PORT=2500",
-			"GOTRUE_SMTP_USER=GOTRUE_SMTP_USER",
+			// NOTE: Needs to be empty string to bypass `unencrypted connection` error
+			// See https://github.com/go-gomail/gomail/issues/113
+			"GOTRUE_SMTP_USER=",
 			"GOTRUE_SMTP_PASS=GOTRUE_SMTP_PASS",
```

Comment:
also seems to work if I removed both env var completely

Reply:
hmm yeah that seems to work